packages/react-components/react-breadcrumb/docs/Spec.md

```diff
@@ -2,54 +2,516 @@
+  { text: 'This is non-clickable folder 4', key: 'f4' },
+  { text: 'This is folder 5', key: 'f5', onClick: _onBreadcrumbItemClicked, isCurrentItem: true },
+];
+
+export const BreadcrumbStaticExample: React.FunctionComponent = () => {
+  return (
+    <Breadcrumb
+      items={items}
+      maxDisplayedItems={3}
+      ariaLabel="Breadcrumb with static width"
+      overflowAriaLabel="More items"
+    />
+  );
+};
+
+function _onBreadcrumbItemClicked(ev: React.MouseEvent<HTMLElement>, item: IBreadcrumbItem): void {
+  console.log(`Breadcrumb item with key "${item.key}" has been clicked.`);
+}
+```
+
+### Northstar (v0)
+
+```jsx
+import { Breadcrumb } from '@fluentui/react-northstar';
+import { ChevronEndMediumIcon } from '@fluentui/react-icons-northstar';
+
+const BreadcrumbExampleIconDivider = props => (
+  <Breadcrumb aria-label="breadcrumb">
+    <Breadcrumb.Item>
+      <Breadcrumb.Link href="">Home</Breadcrumb.Link>
+    </Breadcrumb.Item>
+    <Breadcrumb.Divider>
+      <ChevronEndMediumIcon />
+    </Breadcrumb.Divider>
+    <Breadcrumb.Item>
+      <Breadcrumb.Link href="">Store</Breadcrumb.Link>
+    </Breadcrumb.Item>
+    <Breadcrumb.Divider>
+      <ChevronEndMediumIcon />
+    </Breadcrumb.Divider>
+    <Breadcrumb.Item aria-current="page">T-shirt</Breadcrumb.Item>
+  </Breadcrumb>
+);
+```
 
 ## Prior Art
 
-_Include background research done for this component_
+- [Open UI research](https://open-ui.org/components/breadcrumb)
+- [Convergence epic](https://github.com/microsoft/fluentui/issues/26480)
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### Comparison of [Fabric Breadcrumb](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb) and [Northstar Breadcrumb](https://fluentsite.z22.web.core.windows.net/0.57.0/components/breadcrumb/definition)
+
+### Components
+
+| Purpose                                                                | Fabric     | Northstar         | Matching? |
+| ---------------------------------------------------------------------- | ---------- | ----------------- | --------- |
+| Breadcrumb is a component that indicates the path of the current page  | Breadcrumb | Breadcrumb        | ⚠️        |
+| BreadcrumbItem an actionable item within a Breadcrumb                  |            | BreadcrumbItem    | ❌        |
+| BreadcrumbDivider divides BreadcrumbItem components within Breadcrumb  |            | BreadcrumbDivider | ❌        |
+| An BreadcrumbLink represents a anchor to be used inside the Breadcrumb |            | BreadcrumbLink    | ❌        |
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+By default BreadcrumbButton should be used.
+
+```jsx
+const BreadcrumbV9Example = props => (
+  <Breadcrumb aria-label="breadcrumb">
+    <BreadcrumbItem onClick={() => {}}>
```

Comment:
Later on you have the `onClick` handler on the button itself which is what I think you want here as well.

```suggestion
    <BreadcrumbItem>
```

Reply:
Yeah, you're right. Thank you!